tools/cr/plaster.py

```diff
@@ -258,7 +258,7 @@ def apply(self, dry_run=False):
         for substitution in plaster_file.get('substitution', []):
             pattern = substitution.get('re_pattern', '')
             replace = substitution.get('replace', '')
-            count = substitution.get('count', 0)
+            count = substitution.get('count', 1)
```

Comment:
Does this mean if count is not specified we default to only the first match?

Reply:
what I worry about is someone not specifying this and having a single match, but then there's some change to the upstream file and it gets applied somewhere that it shouldn't. I think in those cases it's better to verify that it's correct. In cases where we know we want to replace all instances we can specify `-1`